transform/mattermost-analytics/models/marts/marketing/_marketing__models.yml

```diff
@@ -0,0 +1,27 @@
+version: 2
+
+models:
+  - name: fct_marketing_campaign_member
+    description: |
+      Marketing data coming from cws, to sync with leads & campaign member in Salesforce.
+
+    columns:
+      - name: marketing_id
+        description: Marketing ID from cws.
```

Comment:
what is a "marketing id"? If we need Nick's input/explanation, let's make a note of this and make sure we document it somewhere.

Reply:
Assuming you're referring to the `id` column in the CWS data - it's just a unique identifier for the row

Reply:
@sri-byte , let's include Nick's explanation that marketing id has no particular meaning - "unique identifier for the row"

Reply:
Updated descriptions.